shared/reducers/chat2.js

```diff
@@ -126,6 +127,19 @@ const metaMapReducer = (metaMap, action) => {
       return ['inboxSyncedClear', 'leftAConversation'].includes(action.payload.reason)
         ? metaMap.clear()
         : metaMap
+    case Chat2Gen.updateConvRetentionPolicy:
+      const update = action.payload.update
+      if (!update.conv) {
+        logger.warn('Got conv retentionpolicy update with no inboxUIItem')
+        return metaMap
+      }
+      const newMeta = Constants.inboxUIItemToConversationMeta(update.conv)
+      if (!newMeta) {
+        // public conversation, do nothing
```

Comment:
this is not the only reason why newMeta can be null, also i don't think you need to enumerate that here